### PR TITLE
Create reference management models template

### PR DESCRIPTION
**Pull request by erikjanwestendorp**

### CODE REVIEW

**Comment by busrasengul** [comment]
Another one to check! You're on fire @erikjanwestendorp 🙌

**Comment by busrasengul** [comment]
Hey @erikjanwestendorp 

Is this for Language file? 😕

**Comment by erikjanwestendorp** [comment]
@busrasengul Uhm.. I think I wasn't fully awake yet.. 😢. Of course it is not for a language file but for a template.. Fixed it in my latest commit 😄

File: Reference/Management/Models/Template.md [hunk 1]
```diff
@@ -1,16 +1,13 @@
 ---
-versionFrom: 6.0.0
-needsV8Update: "true"
+versionFrom: 8.0.0
 ---
 
-# Template
+# Language
```
Comment (busrasengul) [inline Reference/Management/Models/Template.md:5]: This still says Language

File: Reference/Management/Models/Template.md [hunk 2]
```diff
@@ -22,6 +19,70 @@ All samples in this document will require the following using statement:
...
+Gets the Name of the File including extension.
+
+```csharp
+var template = new Template("Page", "page");
+return template.Name;
+```
+
 ## Methods
+
+### .SetMasterTemplateITemplate masterTemplate)
```
Comment (busrasengul) [inline Reference/Management/Models/Template.md:75]:
```suggestion
### .SetMasterTemplate(ITemplate masterTemplate)
```

File: Reference/Management/Models/Template.md [hunk 3]
```diff
@@ -22,6 +19,70 @@ All samples in this document will require the following using statement:
...
+
+Sets the master template of the tempalte.
+
+```csharp
+// Cretae a new template
```
Comment (busrasengul) [inline Reference/Management/Models/Template.md:80]:
```suggestion
// Create a new template
```

**Review by busrasengul — CHANGES_REQUESTED** [review]

File: Reference/Management/Models/Template.md (hunk 1 above)
Comment (erikjanwestendorp) [reply Reference/Management/Models/Template.md]: Not my day..

**Pull request merged**